docker-compose.yml

```diff
@@ -12,6 +12,35 @@ services:
     environment:
       - SERVER_PORT=8080
       - SPRING_PROFILES_ACTIVE=dev
+      - OAUTH_ENDPOINT_URL=http://hmpps-auth:8080/auth
+      - PRISON_API_URL=http://prison-api:8080
+
+  prison-api:
```

Comment:
Adding these for good measure, not sure if we're actually using this at the minute though?

Reply:
think it might be difficult to include basm - I've just been running locally against dev